docs/concepts/CLQL.md

```diff
@@ -296,6 +299,8 @@ class(depth = any):
 
 The query above will only return methods of `classA` for which `classB` has a corresponding method.
 
+For the most part, variables defined anywhere in the query can be passed to functions anywhere else in the query. However, variables defined inside an `exclude` block cannot be passed to functions outside that exclude block and vice versa.
```

Comment:
same w include?

Reply:
I didn't write this, just copied it from somewhere else in the docs on suggestion but I will confirm that it wasn't a type originally